index.js

```diff
@@ -44,3 +44,19 @@ exports.decorateBrowserOptions = defaults => Object.assign({}, defaults, {
   transparent: true,
   frame: false
 })
+
+// Removes the redundant space on mac if there is only one tab
+exports.getTabsProps = (parentProps, props) => {
+  if (process.platform === 'darwin') {
+    var classTermsList = document.getElementsByClassName('terms_terms')
+    if (classTermsList.length > 0) {
+      var classTerms = classTermsList[0]
+      if (props.tabs.length <= 1) {
+        classTerms.setAttribute("style", "margin-top: 0")
+      } else {
+        classTerms.setAttribute("style", "")
```

Comment:
Just one nitpick here and in the line 2 above: Can you use single quotes instead of double quotes?